ingestion/src/main/java/feast/ingestion/options/ImportOptions.java

```diff
@@ -24,8 +24,11 @@
 import org.apache.beam.sdk.options.PipelineOptions;
 import org.apache.beam.sdk.options.Validation.Required;
 
-/** Options passed to Beam to influence the job's execution environment */
+/**
+ * Options passed to Beam to influence the job's execution environment
```

Comment:
I see that indentation has changed in a lot of places. Most of the Java PRs are now having this effect. Are you running spotless or something else?

Reply:
I only used the Google java format options but not the spotless. 
I've applied the spotless in newer commits.